Jenkinsfile

```diff
@@ -104,14 +129,20 @@ pipeline {
 				}
 			}
 		}
-		// This stage archives all logs and database backups produced by UpdateStableIdentifiers
+		// This stage archives all logs and database backups produced by UpdateStableIdentifiers in the Reactome s3 bucket.
 		stage('Archive logs and backups'){
 			steps{
 				script{
-					sh "mkdir -p archive/${currentRelease}/logs"
-					sh "mv --backup=numbered *_${currentRelease}_*.dump.gz archive/${currentRelease}/"
+					def s3Path = "${env.S3_RELEASE_DIRECTORY_URL}/${currentRelease}/update_stable_ids"
+					sh "mkdir -p databases/"
+					sh "mv --backup=numbered *_${currentRelease}_*.dump.gz databases/"
+					sh "mv data-release-pipeline/ortho-stable-id-history/logs/* logs/"
+					sh "rm -rf data-release-pipeline*"
 					sh "gzip logs/*"
-					sh "mv logs/* archive/${currentRelease}/logs/"
+					sh "aws s3 --no-progress --recursive cp databases/ $s3Path/databases/"
+					sh "aws s3 --no-progress --recursive cp logs/ $s3Path/logs/"
+					sh "rm -r databases"
```

Comment:
If the S3 upload fails, I imagine the `rm` steps _will_ be blocked? Has this been tested? It would be a shame to have an S3 failure in a Release, and then lose the files that were supposed to be backed up.

Reply:
Yes, if any error occurs while interacting with the s3 bucket, it will terminate the entire Jenkins process. For example, I modified the $s3Path variable in a test run so that it drew from `s3://eactome` instead of `s3://reactome`, and I got the following error message followed by termination:

```
+ aws s3 --no-progress cp s3://eactome/private/releases/71/update_stable_ids/databases/slice_test_71_snapshot.dump.gz .
fatal error: An error occurred (404) when calling the HeadObject operation: Key "private/releases/71/update_stable_ids/databases/slice_test_71_snapshot.dump.gz" does not exist
[Pipeline] }
[Pipeline] // withCredentials
[Pipeline] }
[Pipeline] // script
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Setup: Back up Curator gk_central DB)
Stage "Setup: Back up Curator gk_central DB" skipped due to earlier failure(s)
```

Reply:
Ok, so we don't need to worry about losing logs to `rm` if the `aws s3` fails earlier.